topics/computational-chemistry/tutorials/cheminformatics/tutorial.md

```diff
@@ -260,15 +254,45 @@ Further, docking requires the coordinates of a binding site to be defined. Effec
 >    - Leave all other options unchanged.
 >    - Rename to 'Prepared ligands'
 > 3. **Calculate the box parameters for an AutoDock Vina job** {% icon tool %} with the following parameters:
->    - {% icon param-file %} *"Input ligand"*: 'Ligand (MOL) file.
+>    - {% icon param-file %} *"Input ligand or pocket"*: `Ligand (MOL)` file.
 >    - {% icon param-file %} *"x-axis buffer"*: `5`
 >    - {% icon param-file %} *"y-axis buffer"*: `5`
 >    - {% icon param-file %} *"z-axis buffer"*: `5`
 >    - {% icon param-file %} *"Exhaustiveness"*: `1`
 >    - The 'Random seed' parameter should be left empty.
->    - Rename to 'Docking config file'
+>    - Rename to 'Docking config file'.
 {: .hands_on}
 
+Perhaps you are interested in a system which does not have a ligand within the binding site (an apoprotein). In this case you need to run the fpocket tool to identify potential binding sites in the protein structure - take a look at the following (optional) section.
+
+
+> ### {% icon details %} How to find the binding site of an apoprotein?
+>
+> If the structure contains no ligand in complex with the protein (i.e. apoprotein), there is an additional step of identifying the binding site. Software is available for automatic detection of pockets which may be promising candidates for ligand binding sites. For example, let's try out the fpocket tool ({% cite LeGuilloux2009 %}) on the Hsp90 structure, imagining we don't have access to the file containing the ligand coordinates.
+> > ### {% icon hands_on %} Hands-on: Finding binding sites using fpocket
+> >
+> > 1. **fpocket** {% icon tool %} with the following parameters:
```

Comment:
Do you want to use the new tool linking syntax here? (I know it isn't throughout the tutorial yet, but, in can be incremental?)

[syntax guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links)